config/webhook/webhook_patch.yaml

```diff
@@ -16,5 +16,12 @@ webhooks:
     timeoutSeconds: 5
     namespaceSelector:
       matchExpressions:
+        # using the control-plane label to bypass Gatekeeper is deprecated
```

Comment:
Per our discussion on slack, to minimize the user impact of removing the `control-plane` label, let's not deprecate the `control-plane` label from the default deployment yaml until we have a better solution.

Reply:
Removed the deadline for deprecation.